docs/draft/6-payloads.md

```diff
@@ -191,6 +199,49 @@ message ImageMessage {
 }
 ```
 
+##### EmojiReaction content type
+
+`EmojiReaction`s represents a user's "reaction" to a specific chat message. For more information about the concept of
+emoji reactions see [Facebook Reactions](https://en.wikipedia.org/wiki/Facebook_like_button#Use_on_Facebook).
+
+This specification RECOMMENDS that the UI/UX implementation of sending `EmojiReactions` requires only a single click
+operation, as users have an expectation that emoji reactions are effortless and simple to perform.  
+
+A `ChatMessage` with `EMOJI_REACTION` `Content/Type` MUST also specify the `EmojiReaction` fields `message_id` and `type`
+
+```protobuf
+message EmojiReaction {
+  string message_id = 1;
+  Type type = 2;
+
+  enum Type {
+    UNKNOWN_EMOJI_REACTION_TYPE = 0;
+    LOVE = 1;
+    THUMBS_UP = 2;
+    THUMBS_DOWN = 3;
+    LAUGH = 4;
+    SAD = 5;
+    ANGRY = 6;
+  }
+}
+```
+
+#####  EmojiReactionRetraction content type
+
+`EmojiReactionRetraction`s represent a user removing a reaction, they previously performed, from a specific chat message.
+
+This specification RECOMMENDS that the UI/UX implementation of sending `EmojiReactionRetraction`s requires only a single
+click operation, as users have an expectation that emoji reaction removals are effortless and simple to perform.  
+
+A `ChatMessage` with `EMOJI_REACTION_RETRACTION` `Content/Type` MUST also specify the `EmojiReactionRetraction` field
+`emoji_reaction_id`
+
+```protobuf
+message EmojiReactionRetraction {
+  string emoji_reaction_id = 1;
```

Comment:
Does `emoji_reaction_id` refer to the corresponding `message_id` from `message EmojiReaction`?

Reply:
Yes.

So, `message EmojiReaction` will generate its own id and the `EmojiReactionRetraction` targets the id of the `EmojiReaction`. This approach will allow messages to have multiple reactions.